.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,41 @@
+name: Bug
+description: Report a bug
+title: "Bug: "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        ⚠️ Your issue will be instantly closed as not planned WITHOUT explanation if:
```

Comment:
Is this an automated process?

Reply:
no 😄 just a warning in place so users don't forget to set the title of the issue and the libevm version. I had personally wayyyy too many issues without a title and/or without a version, which make things pretty horrible to maintain, and that reduced them, but we can remove this for now if you feel it's unecessary. I would recommend not automating in case it's still a serious issue, even without title/version 😮